cosmos/providers/dbt/core/operators_kubernetes.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import annotations
+
+from typing import Sequence
+
+import yaml
+from airflow.providers.cncf.kubernetes.backcompat.backwards_compat_converters import (
+    convert_env_vars,
+)
+from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import (
+    KubernetesPodOperator,
+)
+from airflow.utils.context import Context
+
+from cosmos.providers.dbt.core.operators import DbtBaseOperator
+
+
+class DbtKubernetesBaseOperator(KubernetesPodOperator, DbtBaseOperator):
+    """
+    Executes a dbt core cli command in a Kubernetes Pod.
+
+    """
+
+    template_fields: Sequence[str] = (
+        DbtBaseOperator.template_fields + KubernetesPodOperator.template_fields
+    )
+
+    def __init__(
+        self,
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+
+    def build_env_args(self, env: dict):
+        env_vars_dict = {}
+        for env_var in self.env_vars:
+            env_vars_dict[env_var.name] = env_var.value
```

Comment:
Same here, we can set the DBT_PROFILES_DIR env var.

Reply:
Done